cnf-app-mac-operator/config/rbac/role_binding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: ClusterRoleBinding
```

Comment:
Same here, is it possible to keep using a RoleBinding instead of a ClusterRoleBinding?

Reply:
Same as the Role case.